flytekit/models/core/workflow.py

```diff
@@ -612,10 +614,12 @@ def __init__(
         resources: typing.Optional[Resources],
         extended_resources: typing.Optional[tasks_pb2.ExtendedResources],
         container_image: typing.Optional[str] = None,
+        pod_template: typing.Optional[tasks_pb2.K8sPod] = None,
```

Comment:
The return type should be PodTemplate, right

Reply:
@pingsutw  I didn't create a new proto named `PodTemplate` in `tasks_pb2`, so I used the existing protobuf `K8sPod`.